src/NetAppFiles/NetAppFiles/ActiveDirectory/RemoveNetAppFilesActiveDirectory.cs

```diff
@@ -108,8 +109,16 @@ public override void ExecuteCmdlet()
                 var anfAccount = AzureNetAppFilesManagementClient.Accounts.Get(ResourceGroupName, AccountName);
                 var activeDirectory = anfAccount.ActiveDirectories.FirstOrDefault(a => a.ActiveDirectoryId == ActiveDirectoryId);
                 anfAccount.ActiveDirectories.Remove(activeDirectory);
-                var updatedAnfAccount = AzureNetAppFilesManagementClient.Accounts.BeginCreateOrUpdate(ResourceGroupName, AccountName, anfAccount);
-                success = true;
+                try
+                {
+                    var updatedAnfAccount = AzureNetAppFilesManagementClient.Accounts.BeginCreateOrUpdate(ResourceGroupName, AccountName, anfAccount);
+                    success = true;
+                }
+                catch (ErrorResponseException ex)
+                {
+                    ex = new ErrorResponseException(ex.Body.Error.Message);
```

Comment:
I suggest to add ex as the inner exception
```suggestion
                    ex = new ErrorResponseException(ex.Body.Error.Message, ex);
```